pkg/kubelet/network/cni/cni.go

```diff
@@ -106,10 +106,21 @@ func (plugin *cniNetworkPlugin) SetUpPod(namespace string, name string, id kubec
 	if !ok {
 		return fmt.Errorf("CNI execution called on non-docker runtime")
 	}
-	netns, err := runtime.GetNetNs(id.ContainerID())
+	hostNet, err := runtime.IsUsingHostNetwork(id.ContainerID())
```

Comment:
You said "exec". Why would a GO interface be linked to that?

On Fri, Jan 22, 2016 at 4:29 PM, Prashanth B <notifications@github.com>
wrote:

> In pkg/kubelet/network/cni/cni.go
> <https://github.com/kubernetes/kubernetes/pull/20019#discussion_r50609169>
> :
>
> > @@ -106,10 +106,21 @@ func (plugin *cniNetworkPlugin) SetUpPod(namespace string, name string, id kubec
> >  	if !ok {
> >  		return fmt.Errorf("CNI execution called on non-docker runtime")
> >  	}
> > -	netns, err := runtime.GetNetNs(id.ContainerID())
> > +	hostNet, err := runtime.IsUsingHostNetwork(id.ContainerID())
>
> you can, given the nature of the problem I'd defer the refactor (this is
> an interface shared with exec plugins)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/20019/files#r50609169>.
>